src/dialogIssuesFromSheet.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
```

Comment:
This is the UI for the feature - I suggest leave it alone for now and once we've verified the existing features I'd love your input and review on it.

Reply:
https://github.com/ljay79/jira-tools/issues/136